### PR TITLE
Stix transmission

### PR DESCRIPTION
**Pull request by delliott90**
Adds data source connection functionality to stix-shifter

### CODE REVIEW

**Comment by benjamin-craig** [comment]
Looks good, ran the tests, tested the dummy translate module and tested the bigfix transmission module.

File: main.py [hunk 1]
```diff
@@ -4,58 +4,115 @@
...
+
+    # positional arguments
+    transmit_parser.add_argument(
+        'module', choices=stix_shifter.TRANSMISSION_MODULES,
+        help='choose which connection module to use'
+    )
+    transmit_parser.add_argument(
+        'connection',
+        type=str,
+        help='Data source Host'
```
Comment (emishas) [inline main.py]: could you please fix this help message?
I know it was like this in transmission, my miss

File: main.py [hunk 2]
```diff
@@ -4,58 +4,115 @@
...
+    )
+    transmit_parser.add_argument(
+        'configuration',
+        type=str,
+        help='Data source Port'
```
Comment (emishas) [inline main.py]: same here

File: main.py [hunk 3]
```diff
@@ -4,58 +4,115 @@
...
+        if args.stix_validator:
+            options['stix_validator'] = args.stix_validator
+        if args.data_mapper:
+            options['data_mapper'] = args.data_mapper
+        result = shifter.translate(
+            args.module, args.translate_type, args.data_source, args.data, options=options)
+    elif args.command == TRANSMIT:
+        result = shifter.transmit(args)
+    else:
+        result = "Must call either 'translate' or 'transmit'."
```
Comment (emishas) [inline main.py]: I don't think you need to process this page, 
it should be handled automatically by the argparser

**Pull request merged**